libs/coin-modules/coin-evm/src/api/index.integ.test.ts

```diff
@@ -89,4 +89,40 @@ describe.each([
       });
     });
   });
+
+  describe("estimateFees", () => {
+    it("estimates fees for native asset transfer", async () => {
+      const result: FeeEstimation = await module.estimateFees({
+        type: "send",
+        amount: 100000000000000n, // 0.0001 ETH (smaller amount)
+        sender: "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045",
+        recipient: "0x7b2c7232f9e38f30e2868f0e5bf311cd83554b5a",
+        asset: {
+          type: "native",
+        },
+      });
+
+      expect(result).toHaveProperty("value");
+      expect(typeof result.value).toBe("bigint");
```

Comment:
```suggestion
      expect(result.value).toBeInstanceOf(BigInt);
```
You can give this a try